tests/integration/suite/daprd/subscriptions/declarative/operator/v2alpha1/bulk/http.go

```diff
@@ -16,6 +16,12 @@ package bulk
 import (
 	"context"
 	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	rtv1 "github.com/dapr/dapr/pkg/proto/runtime/v1"
 
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
```

Comment:
Order imports.

Reply:
lint did not catch this.

Reply:
probably in `tests` it does not do it, similar to `make format` does not include tests

Reply:
I'm trying to finish conflict resolution on the buckets pr, and be back to this after